central/notifiers/syslog/tcp_sender.go

```diff
@@ -82,6 +83,14 @@ func validateRemoteConfig(endpointConfig *storage.Syslog_TCPConfig) (string, err
 		return "", errors.Errorf("invalid port number %d must be between 1 and 65535", port)
 	}
 
+	sysURL := fmt.Sprintf("tcp://%s:%d", endpointConfig.GetHostname(), endpointConfig.GetPort())
```

Comment:
There's some duplicated string concat here (and in line 94). Perhaps you can generate `fullHostname` here and return that in line 94?

And in line 88 use that with a Sprintf as well.